docs/content/dev/public_api.yml

```diff
@@ -900,19 +900,21 @@ components:
     MediaUpload:
       type: object
       properties:
-        note:
-          type: string
-          description: ID of the note the file was uploaded to
-        user:
-          type: string
-          description: username of the user who uploaded the file
-        url:
+        link:
           type: string
           description: URL of the file
+        noteId:
```

Comment:
Maybe change this to `owningNote` or `parentNote` or similar?
That would make the object more self-documenting.